internal/system/agent/services.go

```diff
@@ -26,11 +30,15 @@ func getHealth(services []string, registryClient registry.Client) (map[string]in
 		}
 
 		// the registry service returns nil for a healthy service
-		if err := registryClient.IsServiceAvailable(service); err != nil {
+		ok, err := registryClient.IsServiceAvailable(service)
+		if err != nil {
 			health[service] = err.Error()
 			continue
 		}
-
+		if !ok {
+			health[service] = fmt.Sprintf("service %s is not available", service)
+			continue
```

Comment:
Outside the scope of this PR, but looks they are handling the errors via setting health[service].

Reply:
@michaelestrin Is this health[service] the intention to capture all errors instead of returning the error?  This function always returns nil for error.

Reply:
@jim-wang-intel your concern is outside the scope of this PR.  Please raise a separate issue if you want to see this addressed.

Reply:
ok, works for me.  just want to make sure that is addressed. thanks!